ehitchhiking-ui/src/app/shared/modals/profile-modal/cars-info.service.ts

```diff
@@ -11,22 +11,18 @@ export class CarsInfoService {
 			return formBuilder.group({
 				model: [car.model, [Validators.required, Validators.pattern('^[0-9a-zA-Z-]{3,50}$')]],
 				color: [car.color, [Validators.required, Validators.pattern('^[a-zA-Z-]{3,20}$')]],
-				carNumber: [car.carNumber, [Validators.required, Validators.pattern('^[0-9a-zA-Z-]{6,8}$')]],
-				experience: [
-					car.experience,
-					[Validators.required, Validators.pattern('^[0-9]{1,2}$'), Validators.max(80)],
-				],
+				number: [car.number, [Validators.required, Validators.pattern('^[0-9a-zA-Z-]{6,8}$')]],
 			});
 		});
 		// TODO: put object {child-forms: group} into FormGroup
 		return new FormGroup(group as any); // group type is [key:string]: FormGroup
 	}
 
-	getCarsInfo(carsInfoForm: FormGroup, numOfCars: number): Car[] {
+	getCarsInfo(carsInfoForm: FormGroup, numOfCars: number, oldCars: Car[]): Car[] {
 		const cars: Car[] = [];
 		for (let i = 0; i < numOfCars; ++i) {
```

Comment:
pls add comment that it is impossible to use map here and reason why it's impossible

Reply:
added